### PR TITLE
Consolidate Update parts into main update file.

### PR DESCRIPTION
**Pull request by mikelittle**
Change wording around roles screenshot re: Yoast
Format to standards

### CODE REVIEW

File: other-docs/guides/upgrading/v14.md [hunk 1]
```diff
@@ -59,21 +62,21 @@ In your top level project folder, add the `humanmade/workflows` framework
 package
 
 ```sh
-## Add Frameworks package
+# Add Workflows package
```
Comment (roborourke) [inline other-docs/guides/upgrading/v14.md:65]: Wanted to add a quick note this won't be a complete like-for-like swap - this module in particular has a few integrations for other modules like analytics.

Comment (jerico) [reply other-docs/guides/upgrading/v14.md:65]: Should we add a note that there are particular workflow that requires additional modules?

Comment (mikelittle) [reply other-docs/guides/upgrading/v14.md:65]: @roborourke Ping ☝🏽

Comment (roborourke) [reply other-docs/guides/upgrading/v14.md:65]: Sorry @mikelittle - I think if the workflows module is gone, then yeah perhaps updating the analytics module to check for the existence of the Workflows plugin instead is the way to go.

Comment (roborourke) [reply other-docs/guides/upgrading/v14.md:65]: Actually, I suppose we want to be removing any traces of dependency or support for things that are no longer bundled? Maybe the approach is to move the workflows code out of the analytics module to a gist and reference that saying, "If you are using Workflows and Altis Analytics you can find the legacy integration code for analytics notifications here".

What do you think?

Comment (mikelittle) [reply other-docs/guides/upgrading/v14.md:65]: Yep. I can do that.

By the way, is Analytics staying in Altis cloud now? Or is this just until Accelerate replaces it?

Comment (roborourke) [reply other-docs/guides/upgrading/v14.md:65]: I’m pretty it’s staying but deprecated, until removal in v15. I’d check with @veselala

Comment (veselala) [reply other-docs/guides/upgrading/v14.md:65]: as per my last conversation with @rmccue we will bee keeping Accelerate functionalities in Altis till v15, as we need to communicate and migrate our current customers to Accelerate first.

Comment (mikelittle) [reply other-docs/guides/upgrading/v14.md:65]: I have removed the workflow code from Altis/Analytics PR (https://github.com/humanmade/altis-analytics/pull/376)

**Review by roborourke — APPROVED** [review]

**Pull request merged**